bioptim/examples/muscle_driven_ocp/muscle_excitations_tracker.py

```diff
@@ -109,7 +109,7 @@ def generate_data(
 
     def dyn_interface(t, x, u):
         u = np.concatenate([np.zeros(n_tau), u])
-        return np.array(dynamics_func(x, u, [])).squeeze()
+        return np.array(dynamics_func(x, u, [])[:, 0]).squeeze()
```

Comment:
I am not sure why it is not just [0]?

Reply:
nope, I need all the states from the first column. :+1: